### ISSUE TITLE
sort_v problem

### ISSUE THREAD

**Issue by hudongloop** [lilanxiao/Rotated_IoU#4]
hi,

  i‘m interested in your Rotated_IoU, but my area calculation is wrong. When I check the code, sort_v function was wrong. Test as follows(the result of corners are not in order) :
  ```
    v = torch.zeros([1, 1, 24, 2]).float().cuda()
    v_1 = torch.Tensor([[330, 145], [270, 170], [250, 270], [320, 370], [430, 350], [470, 280], [460, 200], [415, 145]]).float().cuda()
    v[:, :, :len(v_1), :] = v_1
    mean = torch.mean(v, dim=2, keepdim=True)
    v = v - mean
    m = torch.zeros([1, 1, 24]).cuda()
    m[:, :, :len(v_1)] = 1
    m = m.bool()
    nv = torch.sum(m.int(), dim=-1).int().cuda()
    result = sort_v(v, m, nv)
  ```

**Comment by hudongloop** [comment]
The mean calculation is error? the mean point are not in polygon inside. When I change:`mean = torch.sum(v, dim=2, keepdim=True).div(len(v_1))`, it work.

**Comment by lilanxiao** [comment]
@hudongloop 
Hi, 
thank you very much for the issue! Yes you are right, it's a bug. 
It's now fixed.

**Comment by lilanxiao** [comment]
Interestingly, the accuracy of the network in the `demo.py` is significantly increased by fixing this bug.
